packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -93,7 +93,14 @@ export const Field = ({
   return (
     <div className={className} data-field-name={name}>
       <label>
-        {label && <span className="label-text">{label}</span>}
+        {label && (
+          <span
+            data-testid="input-label-text"
+            className="font-medium text-sm text-grey-400 block mb-2 text-left"
```

Comment:
Are we sure that all instances that use this field use the same styling, so that we can do this?

Reply:
Those styles are the default for all input labels as per `/packages/fxa-payments-server/src/index.scss` L134:

```
.input row .label-text {
  color: #686869;
  display: block;
  font-size: 13px;
  font-weight: 500;
  margin-bottom: 7px;
  text-align: left;
}
```

There are a few exceptions, namely, `color: #686869; font-size: 13px; margin-bottom: 7px;` From our discussion with @LZoog a few weeks ago and as per our [Design Guide](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-settings/index.html?path=/story/design-guide-settings--introduction), [docs](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings#styling-components), and a11y on form element font sizes (which doesn't have specific minimums as per WCAG, but does recommend going bigger over smaller for safe legibility cross-device and cross-browser), the following adjustments were made:

- As we don't have an equivalent for the colour, I opted to go with [text-grey-400](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-settings/index.html?path=/story/design-guide-settings--colors), which was the closest match
- Between a font-size of 12px or 14px, I found the latter to be more legible so I used `text-sm`
- Similarly, I opted for `mb-2` as the 1px difference is negligible.